src/app/components/SearchResults/ResultsList.jsx

```diff
@@ -44,8 +45,8 @@ class ResultsList extends React.Component {
   }
 }
 
-ResultsList.contextTypes = {
```

Comment:
Ah so this one should be a context type because you call it above as `this.context.router.push...`.